lib/src/base-styles/_nx-global-header.scss

```diff
@@ -7,16 +7,24 @@
 @use '../scss-shared/nx-container-helpers';
 @use '../scss-shared/nx-button-helpers';
 @use '../scss-shared/nx-text-helpers';
+@use '../scss-shared/nx-dark-mode-helpers';
 
 .nx-global-header {
   align-items: center;
-  background-color: var(--nx-swatch-white);
-  border-bottom: var(--nx-border-default);
+  background-color: var(--nx-color-global-header-background);
+  border-bottom: 1px solid;
+  border-color: var(--nx-color-global-header-border);
   display: flex;
   grid-area: header; // see nx-page's grid-template
   height: 42px;
   padding: 0 var(--nx-spacing-6x);
 
+  @include nx-dark-mode-helpers.dark-mode {
+    background-color: var(--nx-color-global-header-background);
```

Comment:
you should be able to get rid of the background-color & border-color here. _nx-colors.scss will take care of it!

Reply:
> you should be able to get rid of the `background-color` & `border-color` here.` _nx-colors.scss` will take care of it!

Ops forgot to remove that! Thanks!
removed it here: 3d943e5eb19da2a899947b4772f6da631aad7d80

Reply:
> > you should be able to get rid of the `background-color` & `border-color` here.` _nx-colors.scss` will take care of it!
> 
> Ops forgot to remove that! Thanks! removed it here: [3d943e5](https://github.com/sonatype/sonatype-react-shared-components/commit/3d943e5eb19da2a899947b4772f6da631aad7d80)

You should be able to remove the `border-color` as well from the dark-mode helper

Reply:
> > > you should be able to get rid of the `background-color` & `border-color` here.` _nx-colors.scss` will take care of it!
> > 
> > 
> > Ops forgot to remove that! Thanks! removed it here: [3d943e5](https://github.com/sonatype/sonatype-react-shared-components/commit/3d943e5eb19da2a899947b4772f6da631aad7d80)
> 
> You should be able to remove the `border-color` as well from the dark-mode helper

Ops, here it is: 64be5391553bf7eb61523602922953160dba908b